Src/ReleaseHistory.md

```diff
@@ -19,6 +19,7 @@
 ## Unreleased
 
 - Bump MongoDB.Driver from 2.13.1 to 2.15.0 and Microsoft.AspNetCore.Http from 2.1.0 to 2.2.0. [#608](https://github.com/microsoft/sarif-pattern-matcher/pull/608)
+- Bump SARIF.SDK from 2.4.13 to 2.4.15. [#612](https://github.com/microsoft/sarif-pattern-matcher/pull/612)
```

Comment:
>Bump SARIF.SDK from 2.4.13 to 2.4.15.  [](http://example.com/codeflow?start=2&length=38)

Bump Sarif.Sdk from 2.4.13 to 2.4.15 by updating submodule to commit XXXXXXXXXXXXXXX.